docs/platforms/dotnet/common/configuration/filtering.mdx

```diff
@@ -12,7 +12,7 @@ We also offer [Inbound Filters](/concepts/data-management/filtering/) to filter
 
 ## Filtering Error Events
 
-Configure your SDK to filter error events by using the <PlatformIdentifier name="before-send" /> callback method and configuring, enabling, or disabling integrations.
+To prevent certain errors from being reported to Sentry, use the <PlatformIdentifier name="before-send" /> or <PlatformIdentifier name="add-exception-filter" /> configuration options, which allowes you to evaluate whether to send an error or now. Alternatively, you can also control the behaviour by enabling, or disabling integrations.
```

Comment:
```suggestion
To prevent certain errors from being reported to Sentry, use the <PlatformIdentifier name="before-send" /> or <PlatformIdentifier name="add-exception-filter" /> configuration options, which allows you to evaluate whether to send an error or now. Alternatively, you can also control the behaviour by enabling, or disabling integrations.
```